docs/installation/presto-admin-installation.rst

```diff
@@ -5,19 +5,42 @@ Installing Presto Admin
 =======================
 *Prerequisites:* `Python 2.6 or Python 2.7 <https://www.python.org/downloads>`_
 
+To install ``presto-admin``:
+
+Presto Admin is packaged as an offline installer --
+``prestoadmin-<version>-offline.tar.bz2`` -- and as an online
+installer -- ``prestoadmin-<version>-online.tar.bz2``. The offline
+installer includes all of the dependencies for ``presto-admin``, so it
+can be used on a cluster without an outside network connection. The
+offline installer is currently only supported on RedHat Linux 6.x or
+CentOS equivalent.
+
+The online installer downloads all of the dependencies when you call
```

Comment:
Suggest: The online installer downloads all of presto-admin's dependencies when you run